.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 4
+
```

Comment:
I also like `trim_trailing_whitespace = true`, perhaps we should add it (and also remove all the trailing whitespace in the current sources?)?